doc/quickstart/rasdaman_quickstart.rst

```diff
@@ -34,24 +35,24 @@ which will launch a local demonstrator in the browser. For a more hands-on
 tutorial though, continue with the next examples here.
 
 
-Running WCPS Queries
-================================================================================
+Running WCPS queries
+====================
 
 Either:
   * Go to the `rasdaman web client <http://localhost:8080/rasdaman/ows>`_, switch to the *ProcessCoverages* tab, and enter your query in the query boxes and press send.
   * Add your query in this url: http://localhost:8080/rasdaman/ows?query=your_query and run it using a browser.
 
 
-Running Coverage
-================================================================================
+Running coverage
+================
 
-The example coverage which will be used in the seqeul is a 6 MB image as bellow:
+The example coverage which will be used in the query is a 6 MB image as below:
```

Comment:
"in the seqeul" should be changed to "subsequently"

Reply:
Hurrah, thank you! I did wonder about that. Will fix.